README.md

```diff
@@ -1,17 +1,23 @@
 # Samvera CircleCI Orb
 
-The Samvera CircleCI Orb is meant to make testing Samvera and Samvera-based projects easier.
+Code: [![Samvera Core Component](https://img.shields.io/badge/samvera-core--component-brightgreen)](https://github.com/samvera/maintenance#samvera-core-components)
+[![Build Status](https://circleci.com/gh/samvera/samvera-circleci-orb.svg?style=svg)](https://circleci.com/gh/samvera/samvera-circleci-orb)
+[![Version](https://badges.circleci.com/orbs/samvera/circleci-orb.svg)](https://circleci.com/developer/orbs/orb/samvera/circleci-orb?version=1.0.3)
```

Comment:
I think it might be best to drop the version from the link so it will always go to the latest (which should match with the badge then).
```suggestion
[![Version](https://badges.circleci.com/orbs/samvera/circleci-orb.svg)](https://circleci.com/developer/orbs/orb/samvera/circleci-orb)
```